src/main/java/com/amalitech/gpuconfigurator/model/attributes/AttributeOption.java

```diff
@@ -0,0 +1,42 @@
+package com.amalitech.gpuconfigurator.model.attributes;
+
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "attributeOptions")
```

Comment:
The convention is to use snake `attribute_options` if you're opting for camelCase, then you need to be consistent across the entire project. I'd recommend to go with the convention though.

Reply:
done